lib/sse_test_server/sse_server.ex

```diff
@@ -76,59 +76,40 @@ defmodule SSETestServer.SSEServer do
     Process.flag(:trap_exit, true)
     listener_ref = make_ref()
     ranch_args = args |> Enum.filter(fn {k, _} -> k in [:port] end)
-    dispatch = :cowboy_router.compile([{:_, handlers_for_endpoints()}])
+    dispatch = :cowboy_router.compile(
+      [{:_, [
```

Comment:
`:_`? 😿

Reply:
It's the atom `_` (spelled `'_'` in Erlang, `:_` or `:"_"` in Elixir) and cowboy uses it as a token to indicate "accept any hostname/path/whatever".

Reply:
So this is any host and any path?

Reply:
Yes -- this is how the fallback handler for creating new endpoints worked previously.